airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml

```diff
@@ -20,9 +20,8 @@ deepdiff = "^6.3.0"
 mergedeep = "^1.3.4"
 pydash = "^6.0.2"
 dpath = "^2.1.5"
+dagster-cloud = "^1.7.5"
```

Comment:
is this a dependency of `dagster-cloud[serverless]`? Wondering why we've added it